### ISSUE TITLE
GF ticks: possible solutions

### ISSUE THREAD

**Issue by repied** [repied/tables#59]
Neural network and random forest predicting `NDL_limit = f( GFl ,GFh, O2, depth, stop)`
- try a validation set  to see if there is really some overfitting
- take one and use it on the site , even if that's not accurate
- push the lookup solution on a 10^n grid in another branch

i feel that rnmse of 30 mins can be beaten, this is so large!
There is more structure as the function is continuous, maybe need to beak it down to a combination of univaraite functions . Or fit different models for different stops

What about skipping the ml entierly: just in the javascript, when we have a state (GFl ,GFh, O2, depth), we sample 20 points of the duration to generate 20  real plans and find the stops NDL. Maybe 20 is not enough
